internal/pkg/mock/config.go

```diff
@@ -0,0 +1,17 @@
+package mock
```

Comment:
I'm a little confused with config as a package with functions, as essentially we want to read from a config file (using viper) and store the value in a struct. 

Mocking the config for SetCommand becomes

```
type SetCommand struct {
	Config prm.Config
}

cmd SetCommand {
  Config {
    puppetVersion  {
      major: 7,
      minor: 0,
      patch: 0,
    } ,
	  backend       "DOCKER"
    }
  }
}
```